### ISSUE TITLE
How to run Inference and Fine-Tuning on Instance Segmentation EVA-02?

### ISSUE THREAD

**Issue by FrancoisPorcher** [baaivision/EVA#127]
Hi EVA team!

Thank you for your great work. I am using the EVA-02 model fine tuned in o365 and LVIS. 

Could you please write some examples on:
- How to run the inference with a set of images (Instance Segmentation)
- How to fine-tune the model on new data? What should be the format?
- Could you also please provide guidance on the parameters to change (learning rate, batch, freeze the encoder? freeze it partially? not at all?). 
- Where should we change the loss functions (for Cascade Mask RCNN)

That would be of great help for everyone! Thanks

More details about the model I am using:
- config: projects/ViTDet/configs/eva2_o365_to_lvis/eva2_o365_to_lvis_cascade_mask_rcnn_vitdet_l_8attn_1536_lrd0p8.py
- checkpoint: checkpoints/eva02_L_lvis_sys_o365.pth
